sdk/inc/azure/core/az_context.h

```diff
@@ -52,24 +52,24 @@ struct az_context
 extern az_context az_context_app;
 
 /**
- * @brief az_context_with_expiration creates a newexpiring az_context node that is a child of the
- * specified parent.
+ * @brief az_context_create_with_expiration creates a newexpiring az_context node that is a child of
```

Comment:
I just noticed this while doing a repo-wide/docs spell check. The comment is weirdly worded and should be fixed. For example, what is `newexpiring`?

Reply:
@RickWinter , what should we rename it to?

Reply:
It's fine to do this separately :)

Reply:
I think it is meant to be `new expiring`, that makes sense, I'll do the update.